### DISCUSSION TITLE
[求助]是否有方法让pcl2的缓存路径保存在当前文件夹下

### DISCUSSION THREAD

**Discussion by kaseeen** [Meloong-Git/PCL#3917] [提问] [answered]
家里的电脑和学校电脑盘符不一样 设置了路径没法同步 ~总是缓存文件到处乱飞~ 
有办法让缓存文件夹保存在当前目录下吗

**Comment by zkitefly** [comment]
意思是支持相对路径的设置？

**Reply by kaseeen** [reply]
emmm 差不多吧 能达成这个效果吗

**Reply by kong-que-dnf123** [reply, marked as answer]
>emmm 差不多吧 能达成这个效果吗

十分抱歉 但是相对路径希望渺茫...

![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/9d011686-6514-41b0-81a9-356d363945e4)

**Reply by wuliaodexiaoluo** [reply]
> emmm 差不多吧 能达成这个效果吗

龙猫拒了

**Reply by kaseeen** [reply]
啊这 那有别的方法可能达成类似效果吗😭

**Reply by wuliaodexiaoluo** [reply]
> 啊这 那有别的方法可能达成类似效果吗😭

猜你可能需要：Microsoft Windows 虚拟磁盘

**Reply by Silverteal** [reply]
> 啊这 那有别的方法可能达成类似效果吗😭

我能想象到的相对简单一点的方法是：设置一个主页按钮，通过脚本在不同环境实现本地文件夹-TEMP目录的文件移动。

**Reply by WorldHim** [reply]
> 啊这 那有别的方法可能达成类似效果吗😭

试试把缓存文件夹 软/硬链接 到当前目录

**Reply by Silverteal** [reply]
> 试试把缓存文件夹 软/硬链接 到当前目录

软链接不适用于这个场景啊……没有一个可靠的路径可以用来放置软链接对象
